configs/default/ZEEZ-ZEEZWHOOP.config

```diff
@@ -0,0 +1,84 @@
+# Betaflight / STM32F411 (S411) 4.1.0 Oct 16 2019 / 11:57:34 (c37a7c91a) MSP API: 1.42
+
+board_name ZEEZWHOOP
+manufacturer_id ZEEZ
+
+# resources
+resource MOTOR 1 A00
+resource MOTOR 2 A01
+resource MOTOR 3 B06
+resource MOTOR 4 B07
+resource PPM 1 A03
+resource LED_STRIP 1 A08
+resource SERIAL_TX 1 A09
+resource SERIAL_TX 2 A02
+resource SERIAL_RX 1 A10
+resource SERIAL_RX 2 A03
+resource I2C_SCL 1 B08
+resource I2C_SDA 1 B09
+resource LED 1 C13
+resource SPI_SCK 1 A05
+resource SPI_SCK 3 B03
+resource SPI_MISO 1 A06
+resource SPI_MISO 3 B04
+resource SPI_MOSI 1 A07
+resource SPI_MOSI 3 B05
+resource ADC_BATT 1 B01
+resource ADC_CURR 1 A04
+resource OSD_CS 1 B10
+resource GYRO_EXTI 1 C15
+resource GYRO_CS 1 A15
+
+# timer
+timer A03 AF3
+# pin A03: TIM9 CH2 (AF3)
+timer A00 AF1
+# pin A00: TIM2 CH1 (AF1)
+timer A01 AF1
+# pin A01: TIM2 CH2 (AF1)
+timer B06 AF2
+# pin B06: TIM4 CH1 (AF2)
+timer B07 AF2
+# pin B07: TIM4 CH2 (AF2)
+timer A08 AF1
+# pin A08: TIM1 CH1 (AF1)
+
+# dma
+dma ADC 1 0
+# ADC 1: DMA2 Stream 0 Channel 0
+dma pin A00 0
+# pin A00: DMA1 Stream 5 Channel 3
+dma pin A01 0
+# pin A01: DMA1 Stream 6 Channel 3
+dma pin B06 0
+# pin B06: DMA1 Stream 0 Channel 2
+dma pin B07 0
+# pin B07: DMA1 Stream 3 Channel 2
+dma pin A08 0
+# pin A08: DMA2 Stream 6 Channel 0
+
+# feature
+feature -RX_PARALLEL_PWM
+feature RX_SERIAL
+feature OSD
+
+# serial
+serial 1 64 115200 57600 0 115200
+
+# master
+set mag_hardware = NONE
+set baro_hardware = NONE
+set serialrx_provider = CRSF
+set blackbox_device = NONE
+set dshot_burst = AUTO
+set dshot_bitbang = OFF
+set motor_pwm_protocol = DSHOT600
+set current_meter = NONE
```

Comment:
Please remove the `ADC_CURR` resource definition if this board is to not support a current meter.

Reply:
Ok i will remove the ADC_CURR definition